### PR TITLE
DHCP leases: do not link to WOL page if WOL plugin is missing (2nd attempt)

### PR DESCRIPTION
**Pull request by fraenki**
a follow-up to #1382

### CODE REVIEW

**Comment by fichtner** [comment]
this looks better. sorry for nitpicking, but:

mark the plugin_enabled with a /* XXX refactor plugin detection */

and:

value = boot ? true : false;

equals

value = bool;

**Comment by fichtner** [comment]
I've added a commit, you can now check for

value = count(plugins_scan('wol'));

**Comment by fraenki** [comment]
True ;) Will push a fix that uses the enhanced `plugins_scan()` function.

**Comment by fraenki** [comment]
Done. Please review :)

File: src/www/status_dhcp_leases.php [hunk 1]
```diff
@@ -417,9 +419,13 @@ function remove_duplicate($array, $field)
                   <td><?=$data['int'];?></td>
                   <td><?=$data['ip'];?></td>
                   <td>
+                      <? if ((int)$plugin_installed > 0) { ?>
```
Comment (fichtner) [inline src/www/status_dhcp_leases.php]: $plugin_installed is enough

**Comment by fraenki** [comment]
And now? :)

**Comment by fichtner** [comment]
Merged, thanks! :)

**Pull request merged**